dev/shim/dune

```diff
@@ -0,0 +1,11 @@
+(rule
+ (targets coqtop)
+ (deps
+  (:coqtop %{project_root}/topbin/coqtop_bin.exe)
+  (:prelude %{project_root}/theories/Init/Prelude.vo))
+ (action
+  (with-outputs-to coqtop-prelude
+   (progn
+    (echo "#!/usr/bin/env bash\n")
+    (bash "echo \"$(pwd)/%{coqtop} -coqlib $(pwd)/%{project_root}\"")
```

Comment:
It seems that additional arguments to `coqtop` such as `-R` won't be passed on to the real coqtop.

Reply:
Oh indeed, good point, amended.